recipes/docling/recipe.yaml

```diff
@@ -0,0 +1,80 @@
+context:
+  name: docling
+  version: 2.6.0
+  python_min: "3.10"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/docling-${{ version }}.tar.gz
+  sha256: 282755472a1a1d4a4088d794c6a89f399a5b7fbcea8b5f45012c441670b538d5
+  patches:
+    # lower the scipy lower bounds to match with c-f and avoid conflicts
+    - fix-deps.patch
```

Comment:
This shouldn't be necessary. We're not building against scipy as a library, so we don't have a "pinned" version for it.

Reply:
If I don't then 1.13 is picked:

```
docling 2.6.0 has requirement scipy<2.0.0,>=1.14.1, but you have scipy 1.13.1.
```

Reply:
Here are the details:

```
Error:   × failed to setup test environment: Cannot solve the request because of: The following packages are incompatible
  │ └─ docling ==2.6.0 pyh29332c3_0 cannot be installed because there are no viable options:
  │    ├─ docling 2.6.0 would require
  │    │  ├─ scipy >=1.14.1,<2.0.0, which can be installed with any of the following options:
  │    │  │  └─ scipy 1.14.1 | 1.14.1 would require
  │    │  │     └─ numpy >=1.23.5, which can be installed with any of the following options:
  │    │  │        ├─ numpy 2.1.3
  │    │  │        ├─ numpy 2.0.0 | 2.0.1 | 2.0.2 | 2.1.0 | 2.1.0 | 2.1.1 | 2.1.2
  │    │  │        └─ numpy 1.23.5 | 1.24.0 | 1.24.1 | 1.24.1 | 1.24.2 | 1.24.3 | 1.24.4 | 1.25.0 | 1.25.1 | 1.25.2 | 1.26.0 | 1.26.0 | 1.26.2 | 1.26.3 | 1.26.4
  │    │  └─ deepsearch-glm >=0.26.1,<0.27.0, which cannot be installed because there are no viable options:
  │    │     ├─ deepsearch-glm 0.26.1 | 0.26.1 would require
  │    │     │  └─ fasttext-for-deepsearch-glm >=0.1.0,<0.1.1.0a0, which cannot be installed because there are no viable options:
  │    │     │     ├─ fasttext-for-deepsearch-glm 0.1.0 | 0.1.0 | 0.1.0 would require
  │    │     │     │  ├─ numpy >=1.22.4,<2.0a0, which can be installed with any of the following options:
  │    │     │     │  │  └─ numpy 1.23.0 | 1.23.1 | 1.23.2 | 1.23.3 | 1.23.4 | 1.23.4
  │    │     │     │  └─ numpy 1.22.*, which cannot be installed because there are no viable options:
  │    │     │     │     ├─ numpy 1.22.4, which conflicts with the versions reported above.
  │    │     │     │     ├─ numpy 1.22.4 | 1.22.4 | 1.22.4 | 1.22.4 would require
  │    │     │     │     │  └─ python >=3.8,<3.9.0a0, for which no candidates were found.
  │    │     │     │     ├─ numpy 1.22.0 | 1.22.0 | 1.22.1 | 1.22.2 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3, which conflicts with the versions reported above.
  │    │     │     │     └─ numpy 1.22.0 | 1.22.0 | 1.22.0 | 1.22.0 | 1.22.1 | 1.22.1 | 1.22.2 | 1.22.2 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.23.0 | 1.23.0 | 1.23.0 | 1.23.0 | 1.23.1 |
  │ 1.23.1 | 1.23.1 | 1.23.1 | 1.23.2 | 1.23.2 | 1.23.2 | 1.23.2 | 1.23.3 | 1.23.3 | 1.23.3 | 1.23.3 | 1.23.4 | 1.23.4 | 1.23.4 | 1.23.4 | 1.23.4 | 1.23.4 | 1.23.4 | 1.23.4 | 1.23.4 would require
  │    │     │     │        └─ python >=3.8,<3.9.0a0, for which no candidates were found.
  │    │     │     └─ fasttext-for-deepsearch-glm 0.1.0 | 0.1.0 | 0.1.0 | 0.1.0 | 0.1.0 | 0.1.0 | 0.1.0 | 0.1.0 | 0.1.0 would require
  │    │     │        └─ python 3.9.* *_cpython, for which no candidates were found.
  │    │     └─ deepsearch-glm 0.26.1 | 0.26.1 | 0.26.1 | 0.26.1 | 0.26.1 | 0.26.1 would require
  │    │        └─ python 3.9.* *_cpython, for which no candidates were found.
  │    └─ docling 2.6.0 | 2.6.0 is excluded because due to strict channel priority not using this option from: 'file:///home/conda/staged-recipes/build_artifacts/'
  │ 
  ╰─▶ Cannot solve the request because of: The following packages are incompatible
      └─ docling ==2.6.0 pyh29332c3_0 cannot be installed because there are no viable options:
         ├─ docling 2.6.0 would require
         │  ├─ scipy >=1.14.1,<2.0.0, which can be installed with any of the following options:
         │  │  └─ scipy 1.14.1 | 1.14.1 would require
         │  │     └─ numpy >=1.23.5, which can be installed with any of the following options:
         │  │        ├─ numpy 2.1.3
         │  │        ├─ numpy 2.0.0 | 2.0.1 | 2.0.2 | 2.1.0 | 2.1.0 | 2.1.1 | 2.1.2
         │  │        └─ numpy 1.23.5 | 1.24.0 | 1.24.1 | 1.24.1 | 1.24.2 | 1.24.3 | 1.24.4 | 1.25.0 | 1.25.1 | 1.25.2 | 1.26.0 | 1.26.0 | 1.26.2 | 1.26.3 | 1.26.4
         │  └─ deepsearch-glm >=0.26.1,<0.27.0, which cannot be installed because there are no viable options:
         │     ├─ deepsearch-glm 0.26.1 | 0.26.1 would require
         │     │  └─ fasttext-for-deepsearch-glm >=0.1.0,<0.1.1.0a0, which cannot be installed because there are no viable options:
         │     │     ├─ fasttext-for-deepsearch-glm 0.1.0 | 0.1.0 | 0.1.0 would require
         │     │     │  ├─ numpy >=1.22.4,<2.0a0, which can be installed with any of the following options:
         │     │     │  │  └─ numpy 1.23.0 | 1.23.1 | 1.23.2 | 1.23.3 | 1.23.4 | 1.23.4
         │     │     │  └─ numpy 1.22.*, which cannot be installed because there are no viable options:
         │     │     │     ├─ numpy 1.22.4, which conflicts with the versions reported above.
         │     │     │     ├─ numpy 1.22.4 | 1.22.4 | 1.22.4 | 1.22.4 would require
         │     │     │     │  └─ python >=3.8,<3.9.0a0, for which no candidates were found.
         │     │     │     ├─ numpy 1.22.0 | 1.22.0 | 1.22.1 | 1.22.2 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3, which conflicts with the versions reported above.
         │     │     │     └─ numpy 1.22.0 | 1.22.0 | 1.22.0 | 1.22.0 | 1.22.1 | 1.22.1 | 1.22.2 | 1.22.2 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.22.3 | 1.23.0 | 1.23.0 | 1.23.0 | 1.23.0 | 1.23.1 |
      1.23.1 | 1.23.1 | 1.23.1 | 1.23.2 | 1.23.2 | 1.23.2 | 1.23.2 | 1.23.3 | 1.23.3 | 1.23.3 | 1.23.3 | 1.23.4 | 1.23.4 | 1.23.4 | 1.23.4 | 1.23.4 | 1.23.4 | 1.23.4 | 1.23.4 | 1.23.4 would require
         │     │     │        └─ python >=3.8,<3.9.0a0, for which no candidates were found.
         │     │     └─ fasttext-for-deepsearch-glm 0.1.0 | 0.1.0 | 0.1.0 | 0.1.0 | 0.1.0 | 0.1.0 | 0.1.0 | 0.1.0 | 0.1.0 would require
         │     │        └─ python 3.9.* *_cpython, for which no candidates were found.
         │     └─ deepsearch-glm 0.26.1 | 0.26.1 | 0.26.1 | 0.26.1 | 0.26.1 | 0.26.1 would require
         │        └─ python 3.9.* *_cpython, for which no candidates were found.
         └─ docling 2.6.0 | 2.6.0 is excluded because due to strict channel priority not using this option from: 'file:///home/conda/staged-recipes/build_artifacts/'
      
```

Reply:
Ah, this is another thing where things go haywire because of the lack of numpy 2.0. As soon as `deepsearch-glm` has numpy 2.0-compatible builds, this can be dropped.